blueprints/dumb/files/__test__/components/__name__.spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import <%= pascalEntityName %> from 'components/<%= pascalEntityName %>/<%= pascalEntityName %>'
```

Comment:
whoops that's just a remnant of blueprints I use in another project.  The file paths and whether or not they should be wrapped in folders is all determined by how the blueprints are made so that import is actually wrong it shouldnt have the nested structure. 

It should say:
`import <%= pascalEntityName %> from 'components/<%= pascalEntityName %>'` since in the starter kit dumb components are just nested in the /components folder